app/forms/common/YesNoAnswer.scala

```diff
@@ -16,14 +16,17 @@
 
 package forms.common
 
+import forms.DeclarationPage
 import forms.Mapping.requiredRadio
+import models.DeclarationType.DeclarationType
+import models.viewmodels.TariffContentKey
 import play.api.data.{Form, Forms, Mapping}
 import play.api.libs.json.Json
 import utils.validators.forms.FieldValidator.isContainedIn
 
 case class YesNoAnswer(answer: String)
 
-object YesNoAnswer {
+object YesNoAnswer extends DeclarationPage {
```

Comment:
I don't think extending `DeclarationPage` is necessary in this case.
It'd make sense if it was used in `Navigator`, but I can't find it being used there.

Reply:
Good spot :+1: Done